hrl_rfid/RFIDread read
geometry_msgs/PoseStamped ps_ant_map
geometry_msgs/PoseStamped ps_base_map